sdk/go.mod

```diff
@@ -1,70 +1,5 @@
 module github.com/pulumi/pulumi-synced-folder/sdk
 
-go 1.20
+go 1.23.11
 
-require (
-	github.com/blang/semver v3.5.1+incompatible
```

Comment:
We don't seem to generate this but I think we would want to? tidy cleans this up but I left as a placeholder to get some guidance here

Reply:
Codegen doesn't generate it by default, so I think we want to keep it, just update it. I'd use something like https://github.com/pulumi/pulumi-random/blob/master/sdk/go.mod as reference.